gitapp/lib/user_details/user_repos_list_widget.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:gitapp/constants/constants.dart';
+
+Column userReposListWidget(UserRepo) {
```

Comment:
To the `userReposListWidget` we are passing a single repo,
And to the 
`userReposWidget` we are passing a list. 
Maybe it should be another wa around?

Reply:
We are passing a single repo because this class is creating container widget for one repository.
And we pass the list because ListView.builder is using it to create list of containers of all repositories for one user.
I don't think it should be another way around.